frontend/src/App.tsx

```
@@ -3,9 +3,9 @@ import { BrowserRouter, Navigate, Route, Routes } from 'react-router';
 
 import CoverLetterQnAFriendsPage from '@/pages//CoverLetterQnAFriendsPage';
 import CoverLetterLandingPage from '@/pages/CoverLetterLandingPage';
+import CoverLetterQnAFriendsPage from '@/pages/CoverLetterQnAFriendsPage';
```

Biome (2.3.14)

[error] Shouldn't redeclare 'CoverLetterQnAFriendsPage'. Consider to delete it or rename it.

'CoverLetterQnAFriendsPage' is defined here:


(lint/suspicious/noRedeclare)